versions/1/data_stream/_dev/test/system/spec.yml

```diff
@@ -1,8 +1,35 @@
 spec:
   additionalContents: false
   contents:
-    - description: System test configuration
+    - description: Configuration of test case in yaml format.
       type: file
-      pattern: '^config.yml$'
+      pattern: '^test-[a-z0-9_.-]+-config.yml$'
       contentMediaType: "application/x-yaml"
-      required: false
\ No newline at end of file
+      required: false
+      content:
```

Comment:
I think we should not define the contents just yet. We can define them later but once if we define them in the spec i may be difficult to change the structure without breaking backwards compatibility.